### DISCUSSION TITLE
Divide "address bar" into three independent tools

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1812] [Ideas]
Hello!

Is it possible to divide the "address bar" into three independent tools:

* the address bar of the page you are viewing (shows the page hierarchy);

* a panel of the history of visited pages - which can be attached to the right or left of the main text window;

* history panel of modified pages - now it is in the "Search" section - which can be attached to the right or left of the main text window;

Thank you very much :)

**Comment by Nick222** [comment]
Move to Discussion, pls...

**Reply by introt** [reply]
Done!
